composer.json

```diff
@@ -3,28 +3,29 @@
         "psr-4": {
             "CF\\": "src/"
         }
-    }, 
-    "description": "A Cloudflare plugin for WordPress", 
-    "license": "BSD-3-Clause", 
-    "name": "cloudflare/wordpress-plugin", 
+    },
+    "description": "A Cloudflare plugin for WordPress",
+    "license": "BSD-3-Clause",
+    "name": "cloudflare/wordpress-plugin",
     "require": {
-        "cloudflare/cf-ip-rewrite": "^1.0.0", 
-        "cloudflare/cloudflare-plugin-backend": "^1.1", 
-        "guzzle/guzzle": "~3.9", 
+        "cloudflare/cf-ip-rewrite": "^1.0.0",
+        "cloudflare/cloudflare-plugin-backend": "^1.1",
+        "guzzle/guzzle": "~3.9",
         "symfony/yaml": "~2.6"
-    }, 
+    },
     "require-dev": {
-        "johnkary/phpunit-speedtrap": "^1.0", 
-        "php-mock/php-mock-phpunit": "^1.1", 
-        "phpunit/phpunit": "4.8.*", 
-        "squizlabs/php_codesniffer": "2.*"
-    }, 
+        "johnkary/phpunit-speedtrap": "^1.0",
+        "php-mock/php-mock-phpunit": "^1.1",
+        "phpunit/phpunit": "4.8.*",
+        "squizlabs/php_codesniffer": "2.*",
+        "wimg/php-compatibility": "*",
+        "simplyadmire/composer-plugins" : "@dev"
+    },
     "scripts": {
-        "format": "vendor/bin/phpcs -n --standard=PSR2 --extensions=php,live.php src/", 
-        "post-install-cmd": [
-            "bash git-hooks/setup.sh"
-        ], 
-        "test": "vendor/bin/phpunit"
-    }, 
+        "format": "vendor/bin/phpcs -n --standard=phpcs.xml",
+        "test": "vendor/bin/phpunit",
+        "post-update-cmd": "rm -rf vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility; cp -rp vendor/wimg/php-compatibility vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility",
+        "post-install-cmd": "rm -rf vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility; cp -rp vendor/wimg/php-compatibility vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility"
+    },
     "version": "3.1.1"
```

Comment:
This could be a good addition to composer.json

https://getcomposer.org/doc/04-schema.md#prefer-stable